doc/specs/vulkan/chapters/extensions.txt

```diff
@@ -223,6 +223,16 @@ wrapped by the validation layers.
 See the <<LoaderAndValidationLayers, "`Vulkan Loader Specification and
 Architecture Overview`">> document for additional information.
 
+[NOTE]
+.Note
+====
+The Specification does not currently include necessary extensions in Valid
+Usage statements for individual commands and structures, although it might
+do so in the future.
+It is instead defined in the <<For fundamentals-validusage-extensions,
+Valid Usage for Extensions>> section.
```

Comment:
The source for the spec is intended for use in building both 1.0 and 1.1 specifications.  What you are referring to is only available in the 1.1 spec, but you didn't use an ifdef.

Reply:
Oh, we fixed that.  Originally (when I wrote those) it was just in the 1.1 spec.  I thought that we had published a 1.0 spec (which I was looking at) based on the common branch, but I guess we haven't.  Sorry!

Reply:
@ianelliottus the published 1.0.71 spec is built from master branch along with 1.1.71.

Reply:
@oddhack interesting.  I've been bringing up: https://www.khronos.org/registry/vulkan/specs/1.0-extensions/html/vkspec.html

That is still 1.0.69, and seems based on the old branch.  I just decided to navigate a bit, and found:

- 1.0.72 in: https://www.khronos.org/registry/vulkan/specs/1.0/html/vkspec.html
- 1.0.69 in: https://www.khronos.org/registry/vulkan/specs/1.0-wsi_extensions/html/vkspec.html
- 1.0.69 in: https://www.khronos.org/registry/vulkan/specs/1.0-extensions/html/vkspec.html
- 1.1.72 in: https://www.khronos.org/registry/vulkan/specs/1.1/html/vkspec.html
- 1.1.72 in: https://www.khronos.org/registry/vulkan/specs/1.1-khr-extensions/html/vkspec.html
- 1.1.72 in: https://www.khronos.org/registry/vulkan/specs/1.1-extensions/html/vkspec.html

For some issues/PRs/MRs, I compare the 1.0 and 1.1 specs.  Since I do so much with extensions, I always look at the version with all extensions (with the 1.0 spec being stuck:-).